skiko/src/jvmMain/kotlin/org/jetbrains/skiko/context/ContextHandler.kt

```diff
@@ -0,0 +1,69 @@
+package org.jetbrains.skiko.context
+
+import org.jetbrains.skija.BackendRenderTarget
+import org.jetbrains.skija.Canvas
+import org.jetbrains.skija.DirectContext
+import org.jetbrains.skija.Picture
+import org.jetbrains.skija.Surface
+import org.jetbrains.skiko.GraphicsApi
+import org.jetbrains.skiko.HardwareLayer
+import org.jetbrains.skiko.hostOs
+import org.jetbrains.skiko.OS
+
+internal val renderApi: GraphicsApi by lazy {
+    val environment = System.getenv("SKIKO_RENDER_API")
+    val property = System.getProperty("skiko.renderApi")
+    if (environment != null) {
+        parseRenderApi(environment)
+    } else {
+        parseRenderApi(property)
+    }
+}
+
+private fun parseRenderApi(text: String?): GraphicsApi {
+    when(text) {
+        "RASTER" -> return GraphicsApi.RASTER
+        "OPENGL" -> return GraphicsApi.OPENGL
+        "VULKAN" -> return GraphicsApi.VULKAN
+        "METAL" -> return GraphicsApi.METAL
+        else -> return GraphicsApi.OPENGL
+    }
+}
+
+internal fun createContextHandler(layer: HardwareLayer): ContextHandler {
+    return when (renderApi) {
+        GraphicsApi.RASTER -> RasterContextHandler(layer)
+        GraphicsApi.OPENGL -> OpenGLContextHandler(layer)
+        GraphicsApi.METAL -> MetalContextHandler(layer)
```

Comment:
Shalln't we throw an exception here?

Reply:
I removed the METAL because it is not ready to use yet.